src/components/helper/AxiosHelper.tsx

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios'
+import { TransactionPayload } from '../../lib/types'
+import { baseURL } from '../../config'
+
+const apiEp = baseURL + '/transactions'
+
+//get Authorization header
+const getUserToken = () => {
+  const userToken = sessionStorage.getItem('token')
```

Comment:
Could be simplified to the following:

`const getUserToken = () => sessionStorage.getItem('token')`